src/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
- [optional] please practice displaying your app from the app.js file instead. App.js is your App component, whereas index.js is the "entry point" for your application.